packages/react-inbox/src/reducer.ts

```diff
@@ -29,13 +29,13 @@ const initialState: InboxState = {
 
 export default (state: InboxState = initialState, action) => {
   switch (action.type) {
-  case "inbox/INIT": {
-    return {
-      ...state,
-      ...action.payload,
-      currentTab: action.payload?.tabs?.[0],
-    };
-  }
+    case "inbox/INIT": {
```

Comment:
something seems to be happening here with extra spaces

Reply:
when i rebased the lint changes into my previous branch, i didn't re-run the prettier on the files.  so this changes now applied them